pkg/cli/cmds/root.go

```diff
@@ -93,6 +93,12 @@ var (
 			EnvVar:      "RKE2_AUDIT_POLICY_FILE",
 			Destination: &config.AuditPolicyFile,
 		},
+		&cli.StringFlag{
+			Name:        "pod-security-config-file",
+			Usage:       "(security) Path to the file that defines Pod Security Admission configuration",
+			EnvVar:      "RKE2_POD_SECURITY_CONFIG_FILE",
+			Destination: &config.PodSecurityConfigFile,
+		},
```

Comment:
Sorry about this very picky comment: wouldn't it be more clear if we added `admission` to the name of the variables? I got lost at some point because I was not sure what PodSecurity meant and then realized it was PodSecurityAdmission config file

- pod-security-admission-config-file
- RKE2_POD_SECURITY_ADMISSION_CONFIG_FILE
- &config.PodSecurityAdmissionConfigFile

Perhaps too long?

Reply:
no I think its fine, I will fix it, thanks